test/addons_test.go

```diff
@@ -281,14 +286,18 @@ func testgroup(t *testing.T, groupname string, version string, jobs ...clusterTe
 		}
 
 		t.Logf("determining old and new versions for upgrade testing addon %s", newAddon.GetName())
-		oldVersion, err := semver.Parse(strings.TrimPrefix(oldAddon.GetAnnotations()[constants.AddonRevisionAnnotation], "v"))
```

Comment:
Do we have any protection or failure in the case where we do not have this sent to use, i.e., revision is not included?

Reply:
Yes, the tests will fail because the `semver.Parse()` will fail.

Reply:
Could we upgrade the revision annotation to a mandatory field soon?

Reply:
We should.